.vscode/settings.json

```diff
@@ -20,5 +20,10 @@
     },
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
-    "files.trimTrailingWhitespace": true
+    "files.trimTrailingWhitespace": true,
+    "python.analysis.extraPaths": [
+        "./llvm-project/libcxx/utils",
+        "./llvm-project/llvm/utils/lit",
+        "./tests/utils"

```

Comment:
Is there a reason we're `./`ing these paths? They're relative to the workspace folder anyways.

Reply:
seems like it makes things more explicit, I don't mind this as a style choice

Reply:
I was simply confused about whether `./` was necessary - I had tried a bunch of paths, added `./`, then found something that worked and didn't think to try removing `./`.

Reply:
As everyone has approved, I'd like to avoid resetting testing and land this as-is, but I'll clean up the `./`s in a followup PR since they weren't really an intentional style choice on my part.